packages/block-library/src/post-template/editor.scss

```diff
@@ -1,7 +1,5 @@
 .editor-styles-wrapper {
 	ul.wp-block-post-template {
-		padding-left: 0;
-		margin-left: 0;
 		list-style: none;
```

Comment:
I think we can also delete `list-style: none`, that is, we can delete `editor.scss` itself, because `list-style: none` already exists in `style.scss`.